distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Telemetry.enso

```diff
@@ -0,0 +1,31 @@
+private
+
+import Standard.Base.Data.Dictionary.Dictionary
+import Standard.Base.Nothing.Nothing
+import Standard.Base.Data.Text.Text
+import Standard.Base.Data.Vector.Vector
+
+polyglot java import org.slf4j.LoggerFactory
+
+
+## Schedules a single telemetry message for sending.
+
+   Arguments:
+   - logger_name: Name of the (telemetry) logger
+   - msg: Message. Must not contain colon.
+   - arg_dict: Mapping of argument names to their values.
+log logger_name:Text msg:Text arg_dict:Dictionary -> Nothing =
```

Comment:
Would it not be way more simpler if we avoided use of `Dictionary` and did it [exactly like via SLF4J](https://github.com/enso-org/enso/issues/12464#issuecomment-2713545076)? E.g. the [MyTeleMsg example](https://github.com/enso-org/enso/issues/12464#issuecomment-2713545076) would be called as:
```
Telemetry.log "org.enso.telemetry.TestTelemetry" "MyTeleMsg: my_text={}, my_number={}" [ "Hi!", 42 ]
```
there is no reason for trying to be more fancy and play with `Dictionary`, concatenation, etc. All of that just adds overhead - both in terms of conceptual complexity as well as performance.